Formula/semgrep.rb

```diff
@@ -188,7 +188,6 @@ def install
 
       # Officially suggested workaround for breaking change in setuptools v50.0.0
       # See: https://sourceforge.net/p/ruamel-yaml/tickets/356/
-      # Relevant Issue: https://github.com/pypa/setuptools/issues/2355
```

Comment:
Any code refs for such `brew audit` check failure?

Reply:
https://github.com/Homebrew/brew/pull/14326

Reply:
How should we referenced closed issues that have relevant discussions?